### DISCUSSION TITLE
New or improved tools for session chairs

### DISCUSSION THREAD

**Discussion by linux-lukas** [OpenSlides/OpenSlides#6292] [Ideas]
To what extent could existing tools in OpenSlides for meeting leaders be improved or new ones added?

**Comment by linux-lukas** [comment]
In the past few days I have spoken with people from the voluntary sector, and we have agreed on the following points: 
- Deposit of the rules of procedure, as is already the case with bylaws.

- In the case of points of orders,  it should then be possible to select the relevant paragraph and section directly. The chair of the meeting should be shown a corresponding pop-up in his or her view with the referenced point of order. 
- In the case of points of orders,  it should then be possible to select the relevant paragraph and section directly. The chair of the meeting should be shown a corresponding pop-up in his or her view with the referenced point of order. 

- For motions to adopt bylaws, this should work similarly, with the difference that the corresponding text of the bylaws is integrated into the text field as a quotation with source input.

**Reply by tsiegleauq** [reply]
The 2nd and 3rd point are the same. Perhaps you copied too fast and forgot one?

**Reply by linux-lukas** [reply]
Yes, points 2-3 are identical. It was not meant that way. Thanks for the advice! :)

**Comment by tsiegleauq** [comment]
> To what extent could existing tools in OpenSlides for meeting leaders be improved or new ones added?

As for every other git project, you can implement your desired features and contribute to the project via pull request. That should be the usual GitHub approach 
Alternatively you can ask openslides.com to have a feature implement and maintained for you, of course.

**Reply by linux-lukas** [reply]
Yes, you are basically right. Unfortunately, they are limited to the absolute basics of Python. Maybe I can help out sometime soon.

However, I wanted to share my thoughts, which I had together with friends, with the project.
